keps/sig-instrumentation/20190404-kubernetes-control-plane-metrics-stability.md

```diff
@@ -330,11 +330,11 @@ Alternatively, one lightweight solution which was previously suggested was docum
 
 ### Static Analysis for Validation
 
-Having a set of wrappers in place which allows us to define custom metadata on metrics is quite powerful, since it enables a number of _theorectically possible_ features, like static analysis for verifying metric guarantees during a precommit phase. How we would actually go about doing this is TBD. It is possible to use the metrics registry to output metric descriptions in a separate endpoint; using static analysis we could validate metrics descriptions with our stability rules.
+[_resolved_](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20190605-metrics-validation-and-verification.md)
 
 ### Beta Stability Level
 
-Currently I am inclined to omit the beta stage from metric versioning if only to reduce initial complexity. It may however become more desirable to include this state in a later design/implementation phase.
+[_resolved_](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20191028-metrics-stability-to-beta.md)
```

Comment:
It looks like the link is to this KEP's beta graduation criteria.  I think this bullet point is about adding Beta as a stability level, right?

Reply:
Yeah but we did discuss it at that point (because it's easy to confuse the two things) and no one saw a point to adding a beta stability level, IIRC?

Reply:
Thats entirely possible, but I don't see anything about it in that KEP... Maybe at least add something like "discussed during <link to kep>, and decided it wasn't neccessary"?

Reply:
updated